### DISCUSSION TITLE
v1.10 上线sql页面报错500

### DISCUSSION THREAD

**Discussion by voice-317418365** [hhyo/Archery#2385] [Q&A]
### 重现步骤

<img width="859" alt="企业微信截图_17000345221027" src="https://github.com/hhyo/Archery/assets/35912159/51301cc3-c2d2-49ca-98f4-6a5709589aaa">

部署方式：docker-compose
版本：1.10.0
没改任何配置

### 预期外的结果

<img width="859" alt="企业微信截图_17000345221027" src="https://github.com/hhyo/Archery/assets/35912159/cce98437-7f8c-4b1a-bde2-c8100ab01eb5">

### 版本

1.10.0

### 部署方式

Docker

**Comment by cxheny** [comment]
检查下mysql或者redis连接是否正常，看下docker logs报错信息。
